sources/packages/web/src/views/student/StudentOverawardsBalance.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <student-page-container layout-template="centered" hide-restriction="true">
```

Comment:
The hyperlink is missing as per figma. 
![image](https://user-images.githubusercontent.com/54600590/228378255-fd74d13b-73d2-4140-aab2-4714c534229b.png)